google_guest_agent/network/manager/manager.go

```diff
@@ -253,7 +257,7 @@ func rollbackLeftoverConfigs(ctx context.Context, config *cfg.Sections, mds *met
 
 	for _, svc := range knownNetworkManagers {
 		if err := svc.RollbackNics(ctx, nic); err != nil {
-			logger.Errorf("Failed to rollback primary nic (left over) config for %s: %v", svc.Name(), err)
+			logger.Warningf("Failed to rollback primary nic (left over) config for %s: %v", svc.Name(), err)
```

Comment:
Why are we changing the log level of all these loggings?

Reply:
most of the errors are not real errors since we're doing rollback on all manager's (most of which are not applicable) so logging it at `Error` level might cause confusion and unnecessary concerns so I downgraded.

Reply:
ok